Makefile

```diff
@@ -88,8 +83,10 @@ bin/python:
 	bin/pip install -r requirements-docs.txt
 
 .PHONY: docs-clean
-docs-clean:  ## Clean docs build directory
+docs-clean:  ## Clean docs build directory (and legacy docs installs)
 	cd $(DOCS_DIR) && rm -rf $(BUILDDIR)/
+	rm -rf bin include lib
```

Comment:
This line should be removed because this virtual environment is now used by Sphinx and MyST. Or we can leave it, but you would need to reinstall Sphinx, MyST, and their dependencies. In another PR `requirements-docs.txt` now has the Sphinx and MyST requirements.

Reply:
Actually, I see that it should remain, because we only install requirements when the `bin/python` is missing or has changed. Instead I just updated the help comment in 4fa3b641c37262de2f8352a60fc383c4aad87b7a